### DISCUSSION TITLE
Added new feature: Capture from remote canlogserver

### DISCUSSION THREAD

**Discussion by CCGene** [collin80/SavvyCAN#694] [Show and tell]
Hi all,
I have added a new function to capture remote CAN messages from "canlogserver" (an application of the can-utils utility). This is very useful when working with embedded systems. In fact, can-utils is a set of can utilities included in many yocto distributions.
Once the canlogserver is launched on the embedded system type:

canlogserver can0 -p 1234

you can then hook up SavvyCan remotely (e.g. windows machine) and capture frames sent from the remote embedded system.

If this is of interest, I could share the code but would need to figure out how.

Thank you very much

**Comment by CCGene** [comment]
Does no one have any interest in this new feature?

**Comment by collin80** [comment]
Sorry I hadn't replied before. I'm certainly interested. I didn't know that canlogserver exists so I wrote a python script that does much the same thing but over MQTT. I suppose my solution has the advantage that it can more easily work over the internet but I like the idea of using the existing tools as much as possible.

**Comment by CCGene** [comment]
Thank you for your response.
How do I share the modified code?

thank you very much

**Reply by cschweers** [reply]
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Reply by CCGene** [reply]
Thank you cschweers. Unfortunately, I am not a contributor. Should I make a pull-request from a fork of the project?

**Comment by CCGene** [comment]
Hi all,
I sent the pull request.

Thank you all for your help.
